src/components/ChatList/elements/ChannelContent.style.ts

```diff
@@ -41,6 +41,6 @@ export const channelContentStyles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: normalize(10),
     lineHeight: 14.52,
-    color: COLORS.white
+    color: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color has been changed from `COLORS.white` to `COLORS.almostBlack`. This is a significant change in color scheme. Make sure this change aligns with the overall design and doesn't affect the visibility and contrast of the text against its background.

```diff
-    color: COLORS.white
+    color: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->